shared-bindings/bleio/Peripheral.c

```diff
@@ -68,25 +72,27 @@ static const char default_name[] = "CIRCUITPY";
 //|
 //|    # Create a peripheral and start it up.
 //|    periph = bleio.Peripheral([service])
-//|    periph.start_advertising()
+//|    adv = ServerAdvertisement(periph)
```

Comment:
Where does ServerAdvertisement come from? It's ok to reference the library here.

Reply:
Oops, I forgot an `import`. Fixed.